### DISCUSSION TITLE
短期开发计划 （ 2024 年 3 月）

### DISCUSSION THREAD

**Discussion by cloudwu** [ejoy/ant#106] [Announcements]
2 月计划推迟到 3 月：

* [x] RmlUI 中使用的图片整合到一张大贴图中，支持 Atlas 描述图集。如果将来支持 2d 渲染管线，会将复用图集的功能。
* [ ] 修复编辑器中过去不常用功能因近期引擎重构引入 bug 。（部分完成）
* [ ] 重构后处理中的抗锯齿系统 TAA 。

3 月新计划：

* [x] 改进相同物件的批量渲染。
* [ ] 优化 PC 上 Debug 版本的性能。
* [x] 支持从 gltf 中直接导入非蒙皮动画：https://github.com/ejoy/ant/issues/95
* [ ] 实现一个更好的阴影模糊算法。
* [ ] 基本的 2D 管线。
* [x] 重构 ltask 的任务调度器。

旧计划暂时搁置：

* [ ] RmlUI 完善，特别是 CSS 驱动的动画。

**Comment by constly** [comment]
希望后续能考虑做一个UI编辑器

**Comment by julywind168** [comment]
macos 上的可用性什么时候安排一下～🤔

**Reply by cloudwu** [reply]
mac 现在是什么问题？游戏应当是可以运行的，编辑器没有测试。

**Reply by julywind168** [reply]
<img width="1060" alt="image" src="https://github.com/ejoy/ant/assets/18049268/7c6285c8-2fe3-4e76-a9f8-4103171be713">
编辑器问题1. 一直存在这个 Bad file descriptor 错误

编辑器问题2. 可以看到图标异常的大了

编辑器问题3. 窗口不能最大化
